indent/rust.vim

```diff
@@ -11,16 +11,16 @@ endif
 let b:did_indent = 1
 
 setlocal cindent
-setlocal cinoptions=L0,(0,Ws,J1,j1,m1
-setlocal cinkeys=0{,0},!^F,o,O,0[,0]
+setlocal cinoptions=L0,(0,Ws,J1,j1,m1,(s,m1
```

Comment:
Replace the `(0` with `(s`, rather than adding it on the end. I agree that `(0` should go, that’s an artefact of when visual indentation was preferred in some places; it is no longer used anywhere in the standard formatting guide.

And `m1` is already there.

```suggestion
setlocal cinoptions=L0,(s,Ws,J1,j1,m1
```